src/renderer/browser/util/getRandomIV.js

```diff
@@ -0,0 +1,5 @@
+import { randomBytes } from 'crypto';
```

Comment:
Also this dep doesn't seem to be included in the `package.json`/yarn.lock file

Reply:
This is built into node.